### PR TITLE
Motd

### PR DESCRIPTION
**Pull request by SudoNikon**
Needed to update the MOTD to have security warning in both Prod and QA.

Also changed the word "bro" in Production to "Y'all"

### CODE REVIEW

**Comment by mshepet** [comment]
the security banner doesn't need to be inside the 'if' statement since it's the same for prod/qa

**Comment by SudoNikon** [comment]
I removed it from the IF statement.

File: colors/recipes/default.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#template "/home/#{node[:owner_name]}/.bash_profile" do
+template "/home/mikejones/.bashrc" do
```
Comment (mshepet) [inline colors/recipes/default.rb]: this recipe only works for mikejones?

File: colors/templates/default/bash_profile.erb [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    if [ -f "$HOME/.bashrc" ]; then
+	. "$HOME/.bashrc"
+    fi
+fi
+
+# set PATH so it includes user's private bin if it exists
+if [ -d "$HOME/bin" ] ; then
+    PATH="$HOME/bin:$PATH"
+fi
+mikejones@dog-qa-color:~$ cat .bashrc
```
Comment (mshepet) [inline colors/templates/default/bash_profile.erb]: were you trying to combine two files here?

Comment (SudoNikon) [inline colors/templates/default/bash_profile.erb]: I am not sure why that was in this pull.  This was me attempting to have a script over write my bashrc in my home directory.  

I thought that was on a different branch marked "colors"

File: colors/recipes/default.rb (hunk 1 above)
Comment (SudoNikon) [inline colors/recipes/default.rb]: I was testing something, trying to get a baseline of it working and then I was going to alter it accordingly moving forward.

File: opsworks_stack_state_sync/templates/default/motd.erb [hunk 3]
```diff
@@ -35,7 +48,7 @@ This is a QA server.
 ╚═╝     ╚═╝  ╚═╝ ╚═════╝ ╚═════╝  ╚═════╝  ╚═════╝   ╚═╝   ╚═╝ ╚═════╝ ╚═╝  ╚═══╝      
                                                                                        
                                                                                        
-Be cool bro!                                                                                       
+Be cool Y'all!
```
Comment (shannihilator) [inline opsworks_stack_state_sync/templates/default/motd.erb]: I support this change!

**Pull request closed without merging**